spec/features/new_listing_spec.rb

```diff
@@ -1,16 +1,27 @@
 feature "new listing" do
+
   scenario "adding a new listing that is then visible" do
     add_listing1
     expect(page).to have_content 'testtitle'
     expect(page).to have_content 'test description'
     expect(page).to have_content '100'
     expect(page).not_to have_content 'sinatra'
   end
+
   scenario "can add multiple listings" do
     add_listing1
     add_listing2
     expect(page).to have_content 'testtitle'
     expect(page).to have_content 'testtitle2'
     expect(page).not_to have_content 'sinatra'
   end
+
+  scenario "can add available dates" do
+    add_listing1
+    click_link('testtitle')
+    expect(page).to have_content 'testtitle'
+    expect(page).to have_content '01/01/2021'
```

Comment:
Nothing to change for now, but perhaps a 'nice to have' which we can incorporate into the work plan: not being able to select dates from the past